patches/0038-ipq40xx-Add-support-for-CIG-WF610D-AP.patch

```diff
@@ -0,0 +1,691 @@
+From d84094eed04c84173b5353a1aaac389c3d4d7b2c Mon Sep 17 00:00:00 2001
+From: Chaitanya Godavarthi <chaitanya.kiran@netexperience.com>
+Date: Wed, 4 Nov 2020 22:45:12 -0500
+Subject: [PATCH] ipq40xx: Add support for CIG WF610D AP
+
+Signed-off-by: Chaitanya Godavarthi <chaitanya.kiran@netexperience.com>
+---
+ package/firmware/ipq-wifi/Makefile            |   4 +-
+ .../ipq40xx/base-files/etc/board.d/01_leds    |   5 +
+ .../ipq40xx/base-files/etc/board.d/02_network |   5 +
+ .../etc/hotplug.d/firmware/11-ath10k-caldata  |  18 +
+ .../base-files/lib/upgrade/platform.sh        |  19 +
+ target/linux/ipq40xx/config-4.14              |  22 +-
+ .../boot/dts/qcom-ipq40xx-ap-cig-wf610d.dts   | 391 ++++++++++++++++++
+ target/linux/ipq40xx/image/Makefile           |  14 +
+ .../901-arm-boot-add-dts-files.patch          |   3 +-
+ 9 files changed, 476 insertions(+), 5 deletions(-)
+ create mode 100755 target/linux/ipq40xx/files-4.14/arch/arm/boot/dts/qcom-ipq40xx-ap-cig-wf610d.dts
+
+diff --git a/package/firmware/ipq-wifi/Makefile b/package/firmware/ipq-wifi/Makefile
+index 7bb6b2bf28..05f4cd9548 100644
+--- a/package/firmware/ipq-wifi/Makefile
++++ b/package/firmware/ipq-wifi/Makefile
+@@ -38,7 +38,8 @@ ALLWIFIBOARDS:= \
+ 	linksys_ea8300 \
+ 	qxwlan_e2600ac \
+ 	tp-link_ap2220 \
+-	tp-link_ec420-g1
++	tp-link_ec420-g1 \
++	cig_wf610d
+ 
+ ALLWIFIPACKAGES:=$(foreach BOARD,$(ALLWIFIBOARDS),ipq-wifi-$(BOARD))
+ 
+@@ -116,5 +117,6 @@ $(eval $(call generate-ipq-wifi-package,linksys_ea8300,Linksys EA8300))
+ $(eval $(call generate-ipq-wifi-package,qxwlan_e2600ac,Qxwlan E2600AC))
+ $(eval $(call generate-ipq-wifi-package,tp-link_ap2220,tp-link AP2220))
+ $(eval $(call generate-ipq-wifi-package,tp-link_ec420-g1,tp-link EC420-G1))
++$(eval $(call generate-ipq-wifi-package,cig_wf610d,CIG WF_610D))
+ 
+ $(foreach PACKAGE,$(ALLWIFIPACKAGES),$(eval $(call BuildPackage,$(PACKAGE))))
+diff --git a/target/linux/ipq40xx/base-files/etc/board.d/01_leds b/target/linux/ipq40xx/base-files/etc/board.d/01_leds
+index 5a36b642e9..9d3a51f7ea 100755
+--- a/target/linux/ipq40xx/base-files/etc/board.d/01_leds
++++ b/target/linux/ipq40xx/base-files/etc/board.d/01_leds
+@@ -64,6 +64,11 @@ zyxel,wre6606)
+ 	ucidef_set_led_wlan "wlan2g" "WLAN2G" "${boardname}:green:wlan2g" "phy0tpt"
+ 	ucidef_set_led_wlan "wlan5g" "WLAN5G" "${boardname}:green:wlan5g" "phy1tpt"
+ 	;;
++cig,wf610d)
++	ucidef_set_led_wlan "wlan2g" "WLAN2G" "${boardname}:blue:wlan2g" "phy0tpt"
```

Comment:
this block is identical to the one above, please just ass the cig identifier to the switch clause